indico/modules/rb/templates/emails/reservations/reminders/finishing_bookings.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'rb/emails/base_email.html' %}
+
+{% block header_recipient %}{{ user.first_name }}{% endblock %}
+{% block subject %}Some of your bookings will come to an end soon{% endblock %}
+
+
+{% block body -%}
+    {%- if reservations | length == 1 -%}
+        There is one booking under your name that will come to an end soon:<br>
+    {%- else -%}
+        There are bookings under your name that will soon come to an end:<br>
```

Comment:
Any reason why the messages are different depending on the number of bookings?
"...will soon..." vs "...will come...soon"

Reply:
Probably forgot to fix it since I was modified those messages a few times